src/orchard/java/classpath.clj

```diff
@@ -32,11 +32,8 @@
    (classloaders (context-classloader))))
 
 (defn modifiable-classloader
-  "Returns the highest classloader in the hierarchy that satisfies
-  `dynapath.util/addable-classpath?`, or nil if none do."
   ([^ClassLoader loader]
-   (last (filter dp/addable-classpath?
-                 (classloaders loader))))
+   loader)
```

Comment:
Mark as deprecated, maybe even fail-fast

Reply:
probably fail-fast is risky, similarly to `assert`. An end user could be used the 'deprecated' path unintentionally.